src/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/simple_if.hpp

```diff
@@ -4,30 +4,30 @@
 
 #pragma once
 
-#include <tuple>
 #include <string>
+#include <tuple>
 #include <vector>
 
-#include "shared_test_classes/base/ov_subgraph.hpp"
 #include "shared_test_classes/base/layer_test_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
 
-namespace SubgraphTestsDefinitions {
+namespace ov {
+namespace test {
 
-using SimpleIfParamsTuple = typename std::tuple<
-        std::vector<ov::test::InputShape>,   // Input shapes
-        ov::test::ElementType,               // Network precision
-        bool,                                // If condition
-        std::string                          // Device name
->;
+using SimpleIfParamsTuple = typename std::tuple<std::vector<ov::test::InputShape>,  // Input shapes
+                                                ov::test::ElementType,              // Network precision
+                                                bool,                               // If condition
+                                                std::string                         // Device name
+                                                >;
 
-class SimpleIfTest:
-        public testing::WithParamInterface<SimpleIfParamsTuple>,
-        virtual public ov::test::SubgraphBaseTest {
+class SimpleIfTest : public testing::WithParamInterface<SimpleIfParamsTuple>,
+                     virtual public ov::test::SubgraphBaseTest {
 public:
-    static std::string getTestCaseName(const testing::TestParamInfo<SimpleIfParamsTuple> &obj);
+    static std::string getTestCaseName(const testing::TestParamInfo<SimpleIfParamsTuple>& obj);
+
 protected:
     void SetUp() override;
-    void compare(const std::vector<ov::Tensor> &expected, const std::vector<ov::Tensor> &actual) override;
+    void compare(const std::vector<ov::Tensor>& expected, const std::vector<ov::Tensor>& actual) override;
```

Comment:
move to `compare_map`? The structure is allowed to compare results with different op types (https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/shared_test_classes/src/base/utils/compare_results.cpp)